lib/gateway/pipelines.js

```diff
@@ -237,10 +205,6 @@ function validatePipelinePolicies (policies, config) {
   policies.forEach(policyObj => {
     const policyNames = Object.keys(policyObj);
 
-    if (policyNames.length !== 1) {
-      throw new Error('there should be one and only one policy per policy-object in pipeline configuration');
-    }
-
     if (policiesConfig.indexOf(policyNames[0]) === -1) {
```

Comment:
and pls replace this part with schema

Reply:
I am not completely sure whether this is a possible scenario.

Basically you're asking to validate that field based on an enum whose possible fields aren't part of the schema, but part of the payload itself. (Not sure if clear).

I'll try to research to see if it's possible, but at first glance I do not think it is.

Reply:
ok, we can skip this part
$data seems to be not ready yet https://github.com/json-schema-org/json-schema-spec/issues/331